bouncer/setup_for_test.sh

```diff
@@ -1,6 +1,7 @@
 set -e
 ./commands/setup_vaults.ts
 ./commands/setup_swaps.ts
+./commands/setup_boost_pools.ts
```

Comment:
this is going to increase setup time further - i don't think there's a reason for this, we should run this in parallel with the swaps setup, using Promise.all()